src/engine/shared.rs

```diff
@@ -196,6 +198,14 @@ pub fn validate_name(name: &str) -> StratisResult<()> {
             format!("Name contains control characters : {}", name),
         ));
     }
+    let name_udevregex =
```

Comment:
Use lazy_static trick as described here: https://docs.rs/regex/1.4.1/regex/index.html#example-avoid-compiling-the-same-regex-in-a-loop. We want to make sure that even if this method is called many times, the regular expression is compiled only once.

Reply:
Resolved.